cmd/whitesourceExecuteScan_test.go

```diff
@@ -6,6 +6,7 @@ package cmd
 import (
 	"context"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
Could you please sort imports here. Everything else is alright. Thanks for implementing it.

Reply:
As in 31e04f60641694f0bb41adadf5cb981c89d37a0f or does it need to go further down, before the `"github.com/google/go-github/v45/github"` block?

Reply:
Yeah, like in https://github.com/SAP/jenkins-library/commit/31e04f60641694f0bb41adadf5cb981c89d37a0f will be fine. Thanks